docs/how-to-guides/customise-multipass/set-up-a-graphical-interface.md

```diff
@@ -50,9 +52,78 @@ sudo passwd ubuntu
 
 You will be asked to enter and re-enter a password.
 
-You are done on the server side!
+You are done on the server side! Quit the Ubuntu shell on the running instance with the `exit` command. Proceed to {ref}`connect-to-the-instance-with-an-rdp-client`.
+
+### Set up the GNOME Remote Desktop server
+
+1. Create the following `cloud-init.yaml` configuration file:
+
+   ```{code-block} yaml
+   :caption: cloud-init.yaml
+   :emphasize-lines: 13,14
+
+   package_update: true
+   package_upgrade: true
+   users:
+   - default
+   packages:
+   - ubuntu-desktop
+   - gnome-remote-desktop
+   - winpr3-utils
+   runcmd:
+   - sudo -u gnome-remote-desktop winpr-makecert3 -silent -rdp -path ~gnome-remote-desktop rdp-tls
+   - sudo -u gnome-remote-desktop grdctl --headless rdp set-tls-key ~gnome-remote-desktop/rdp-tls.key
+   - sudo -u gnome-remote-desktop grdctl --headless rdp set-tls-cert ~gnome-remote-desktop/rdp-tls.crt
+   - sudo -u gnome-remote-desktop grdctl --headless rdp set-credentials ubuntu <your-password>
+   - echo ubuntu:<your-password> | sudo chpasswd
+   - echo /usr/sbin/nologin >> /etc/shells
+   - sudo -u gnome-remote-desktop grdctl --headless rdp enable
+   - systemctl --user enable --now gnome-remote-desktop-headless.service
+   - sudo systemctl set-default graphical.target
+   - loginctl enable-linger ubuntu
+   ```
+
+   Replace `<your-password>` with a secure password on the highlighted lines.
+
+1. Launch the configured Multipass instance:
+
+   ```text
+   multipass launch \
+      -vvvv
+      --memory 12G \
+      --cpus=2 \
+      --disk 30G \
+      --timeout 600 \
+      --cloud-init ./cloud-init.yaml 25.10
+   ```
+
+   The command sets the timeout to 10 minutes because the default timeout of 5 minutes isn't enough to initialise the instance.
```

Comment:
I would also comment that the other values can be adjusted (RAM, disk, cores...)